week3_model_free/sarsa.ipynb

```diff
@@ -220,7 +220,7 @@
    "source": [
     "### Cliff World\n",
     "\n",
-    "Let's now see how our algorithm compares against q-learning in case where we force agent to explore all the time.\n",
+    "Let's now see, how our algorithm compares against q-learning in case, where we force an agent to explore all the time.\n",
```

Comment:
Redundant commas?

I'm also not sure about "an" here — it might have to be "the".